index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+        <title>Edwin | Portfolio</title>
+    </head>
+
+    <body>
+        <header>
+            <section class="toolbar">
+                <span class="logo-container">
+                    <h3 class="logo-txt">Edwin G.</h3>
+                </span>
+                <span class="menu-btn">
+                    <img src="./imgs/Union.svg" alt="menu icon">
+                </span>
+            </section>
+        </header>
+        <section class="headline">
+            <span class="primary">
+                Hey There. I'm Edwin
+            </span>
+            <span class="supporting">
+                I’m a Software Developer
+            </span>
+            <span class="description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hesitate to contact me.
+            </span>
```

Comment:
- [ ] This is the title, subtitle, and paragraph and they should be wrapped with tags that are semantically correct for them, like the `h1` tag for the main title, `h2` or `h3` tag for the subtitle, and a `p` tag for a paragraph. Kindly change that.

Reply:
Done.